classifier.py

```diff
@@ -78,20 +78,20 @@ def load_data(train_set):
 		X_data.append(vector)
 		y_data.append(target)
 		if c % 10000 == 0: 
-			print c
+			print(c)
 
-	print len(X_data), 'training examples'
+	print((len(X_data), 'training examples'))
```

Comment:
is there a reason for the double brackets?

Reply:
This is to maintain the exact same output to the console as before, I'm not sure how people are using the script (for example CLI automatons) so I'm not really in any position to make a judgment call about changing the outputs.

This should explain:
```
$ python2
Python 2.7.17 (default, Apr 15 2020, 17:20:14) 
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print(1, 'training examples')
(1, 'training examples')
```
Compared to:
```
$ python3
Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print(1, 'training examples')
1 training examples
```